hibernate-reactive-core/src/main/java/org/hibernate/reactive/event/impl/DefaultReactiveDeleteEventListener.java

```diff
@@ -107,6 +107,14 @@ public void onDelete(DeleteEvent event, Set transientEntities) {
 
 		final EventSource source = event.getSession();
 
+		boolean detached = event.getEntityName() != null
+				? !source.contains( event.getEntityName(), event.getObject() )
```

Comment:
Not sure if we even need to worry about entity names in any of this code, since we don't support them in Hibernate Reactive.